SampleNHAppSwiftUI2/SampleNHAppSwiftUI2/SampleNHAppSwiftUI2App.swift

```diff
@@ -0,0 +1,35 @@
+//
```

Comment:
Same comment as above, please remove all personal information in the headers and use the Microsoft standard:
```
//----------------------------------------------------------------
//  Copyright (c) Microsoft Corporation. All rights reserved.
//----------------------------------------------------------------
```

Reply:
Microsoft standard header applied.